pubspec.yaml

```
@@ -37,6 +37,9 @@ dependencies:
   # Supabase関連
   supabase_flutter: ^2.5.6
   
+  # Google認証
+  google_sign_in: ^6.2.1
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)